tests/README.md

```diff
@@ -95,3 +95,11 @@ via ```./container/build.sh --framework X``` and run via
 The tests will automatically use a local cache at `~/.cache/huggingface` to avoid
 repeated downloads of model files. This cache is shared across test runs to improve performance.
 
+## Running tests locally
+
+To run tests locally, ensure that you have properly setup your environment and have installed the following dependencies in your `venv`:
+
+```bash
+uv pip install pytest-mypy
```

Comment:
is pytest-mypy required - I thought required because of pyproject .... maybe that gets picked up.

another option here is to use the `container/deps/requirements.test` for more complete coverage (until this is moved elsewhere)